blueocean-core-js/src/js/urlconfig.js

```diff
@@ -0,0 +1,36 @@
+let jenkinsRootURL = null;
+let blueoceanAppURL = null;
+
+// General "system" config information.
+//
+// TODO: This should be in a general sharable component.
+// Passing it around in the react context is silly.
+//
+function loadConfig() {
+    const headElement = document.getElementsByTagName('head')[0];
```

Comment:
This code will fail if executed in a unit test environment that uses `Enzyme.mount` (using `jsdom` to mock the browser, but these head tags haven't been set up). I added some error handling to the version in `blueocean-personalization` here:

https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-personalization/src/main/js/config.js